src/matrix/storage/idb/StorageFactory.ts

```diff
@@ -42,6 +42,7 @@ async function requestPersistedStorage(): Promise<boolean> {
             await glob.document.requestStorageAccess();
             return true;
         } catch (err) {
+            console.warn("requestStorageAccess threw an error:", err);
```

Comment:
Ideally we'd use the [logitem object](https://github.com/vector-im/hydrogen-web/blob/master/src/logging/LogItem.ts) in here we use elsewhere rather than the console logs. You can pass it in from StorageFactory.create.